frontend/src/features/properties/filter/PropertyFilterOptions.tsx

```diff
@@ -9,12 +9,12 @@ import { IPropertyFilter } from './IPropertyFilter';
 export const PropertyFilterOptions: React.FC = () => {
```

Comment:
This is what I mentioned in scrum this morning, options is a required field in typeahead so I cannot make these all autocomplete components, I can spend some more time making a component that changes its input type based on the selected option but that will be some more time. For now I have pulled out the property name into a separate input, if that is not ok I will adjust and make new PropertyFilterOptions component to conditionally render the input based on what is selected.